src/angularComponent/ngcOmniboxController.js

```diff
@@ -515,6 +530,26 @@ export default class NgcOmniboxController {
     }
   }
 
+  _updateHint() {
+    this.hint = null;
+
+    const promise = this.inputHint({query: this.query});
+    this._hintFunctionPromise = promise;
+
+    promise && promise.then((hint) => {
```

Comment:
How can `promise` not be defined here? If the app-maker gives us a poor expression that doesn't return a promise, should we not fail fast?